src/System.Net.Sockets/tests/FunctionalTests/SendReceive.cs

```diff
@@ -680,8 +680,8 @@ public void SendRecvPollSync_TcpListener_Socket(IPAddress listenAt, bool pollBef
             }
         }
 
-        [ActiveIssue(16716, TestPlatforms.OSX)] // SendBufferSize = 0 throws
         [Fact]
+        [PlatformSpecific(~TestPlatforms.OSX)] // SendBufferSize = 0 not supported on OSX.
```

Comment:
This test also verifies a receive buffer size of 0.  It'd be good to keep that part of the test on macOS.